src/components/TransactionsTracker/useTransactionsTracker.ts

```diff
@@ -1,40 +1,40 @@
 import { useEffect, useRef } from 'react';
 import { getTransactionsByHashes as defaultGetTxByHash } from 'apiCalls/transactions';
 import { GetTransactionsByHashesType } from 'types';
-import { useCheckTransactionStatus, useGetPendingTransactions } from 'hooks';
-import { getIsTransactionPending } from 'utils';
+import {
+  useCheckTransactionStatus,
+  useGetAccountInfo,
+  useGetNetworkConfig,
+  useRegisterWebsocketListener
+} from 'hooks';
```

Comment:
from hooks/index because in tests jest appends .js extension and might not find hooks